torch_geometric/graphgym/model_builder.py

```diff
@@ -1,22 +1,15 @@
 import time
-import warnings
 from typing import Any, Dict, Tuple
 
 import torch
 
 from torch_geometric.graphgym.config import cfg
+from torch_geometric.graphgym.imports import LightningModule
 from torch_geometric.graphgym.loss import compute_loss
 from torch_geometric.graphgym.models.gnn import GNN
 from torch_geometric.graphgym.optim import create_optimizer, create_scheduler
 from torch_geometric.graphgym.register import network_dict, register_network
 
-try:
-    from pytorch_lightning import LightningModule
-except ImportError:
-    LightningModule = torch.nn.Module
-    warnings.warn("Please install 'pytorch_lightning' for using the GraphGym "
```

Comment:
It seems from below that this won't work with `pytorch_lightning` not installed? In this case does it make more sense to just throw the exception here?

Reply:
I thought about this as well. Might be worth reconsidering the decision of gracefully warning the user whenever the user imports `torch_geometric.graphgym`. We could just error out in that case. We can fix in a follow-up PR if we are all aligned.